### DISCUSSION TITLE
subunits are not fixable in coral

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#1291] [Rigid body modelling]
On 2015-02-04 14:13:25, user tomas wrote:

Dear Saxiers,
I would like to ask you about coral. I am not able to fix subunits. According to coral manual, I shifted domains to origin (0,0,0) but this doesn't help. Coral always dislocated the domains in the working model. I would like to ask you for advice how to overcome this problem.
Thank you

Attachments:

  - [27889_066714dc9b7b220fc6f621845d9cd9a4.zip](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/27889_066714dc9b7b220fc6f621845d9cd9a4.zip)

**Comment by franke-biosaxs** [comment]
On 2015-02-05 15:32:01, user Alex wrote:

Hi
it is not quite clear what you trying to do. in your input pdb files - 2A1I-coral.pdb and ercc1-shift.pdb - are linked by a linker of 4 (actually this is not possible as minimum is 5) but two rigid bodies are too far away from each other in space to link them with only 4 residues. As I understand you have two chains: NTER 9 -2A1I-coral.pdb - LINK 4 - ercc1-shift.pdb and NTER 39 - swiss_model.pdb - LINK 13 - xpf-shift.pdb - CTER 7
So, basically if you know orientation of all available subunits - you need to put them into one PDB file, shift it to origin and then decompose again into individual PDB files. Those PDB files should be fixed when you do modeling.
If you don't know orientation of one chain to another - they should not be fixed. If you know only certain subunits orientation, you can group domains, while modeling ('concerted movement).
HTH, Alex

**Comment by franke-biosaxs** [comment]
On 2015-02-05 15:54:04, user tomas wrote:

Hi Alex,
thank you for your answer and sorry for unclear question. I will explain.
1. linker was set to be 5 residues long - but the proble remain (so no new files are send).
2. problem: I want to fix only ercc1-shift.pdb and xpf-shift.pdb. The orientation is known, domains were together shifted to the origin 0,0,0 - exactly as you suggested, but coral always disconnects them and shift somewhere to 70,30,40. See the log (there is written "subunit was fixed") and the coral model -2.pdb (where is written "New center positioned at  69.536  27.490  40.539").

I would be thankful for any suggestion,
Tomas

**Comment by franke-biosaxs** [comment]
On 2015-02-05 16:12:17, user Alex wrote:

there is no attachment. my gut feeling that it is something trivial. double check the .con file.

**Comment by franke-biosaxs** [comment]
On 2015-02-05 16:48:18, user tomas wrote:

Your gut feeling was right (wrong copy paste in pdb editing). This conversation could be deleted.
Than you,
Tomas
